Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -749,7 +749,7 @@ MenuName="Fliege"
 ; EN: NameArticle=" a "
 NameArticle=" ein "
 ; EN: MenuName="Lesser Brute"
-MenuName="Kleiner Rohling"
+MenuName="Minderer Rohling"
```

Comment:
@SlyMk3 "Lesser" to me has more of an RPG connotation ("Lesser Restoration", "Lesser Demon" etc.), and I think it would be awkward if these were named "Kleinere". They normally use a somewhat more sophisticated term like "Gering", "Minder" etc.